<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="layout">
     <item row="2" column="13">
      <widget class="QLineEdit" name="cloud3"/>
     </item>
     <item row="0" column="12">
      <widget class="QLabel" name="cloud2Label">
       <property name="text">
        <string>云组2</string>
       </property>
       <property name="buddy">
        <cstring>cloud2</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="visLabel">
       <property name="text">
        <string>能见度</string>
       </property>
       <property name="buddy">
        <cstring>vis</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="vis"/>
     </item>
     <item row="2" column="6">
      <widget class="QLineEdit" name="gust"/>
     </item>
     <item row="2" column="12">
      <widget class="QLineEdit" name="cloud2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="name">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(120, 120, 120);</string>
       </property>
       <property name="text">
        <string>TEMPO</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="weatherWithIntensityLabel">
       <property name="text">
        <string>天气现象1</string>
       </property>
       <property name="buddy">
        <cstring>weatherWithIntensity</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="windLabel">
       <property name="text">
        <string>风向风速</string>
       </property>
       <property name="buddy">
        <cstring>wind</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="intervalLabel">
       <property name="text">
        <string>时段</string>
       </property>
       <property name="buddy">
        <cstring>interval</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="wind"/>
     </item>
     <item row="0" column="11">
      <widget class="QLabel" name="cloud1Label">
       <property name="text">
        <string>云组1</string>
       </property>
       <property name="buddy">
        <cstring>cloud1</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QLineEdit" name="cloud1"/>
     </item>
     <item row="0" column="14">
      <widget class="QLabel" name="cbLabel">
       <property name="text">
        <string>积雨云</string>
       </property>
       <property name="buddy">
        <cstring>cb</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="14">
      <widget class="QLineEdit" name="cb"/>
     </item>
     <item row="0" column="13">
      <widget class="QLabel" name="cloud3Label">
       <property name="text">
        <string>云组3</string>
       </property>
       <property name="buddy">
        <cstring>cloud3</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="prob30">
       <property name="text">
        <string>PROB30</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QComboBox" name="weatherWithIntensity"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="interval"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="gustLabel">
       <property name="text">
        <string>阵风</string>
       </property>
       <property name="buddy">
        <cstring>gust</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="prob40">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>PROB40</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QComboBox" name="weather"/>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="weatherLabel">
       <property name="text">
        <string>天气现象2</string>
       </property>
       <property name="buddy">
        <cstring>weather</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>interval</tabstop>
  <tabstop>wind</tabstop>
  <tabstop>gust</tabstop>
  <tabstop>vis</tabstop>
  <tabstop>weatherWithIntensity</tabstop>
  <tabstop>cloud1</tabstop>
  <tabstop>cloud2</tabstop>
  <tabstop>cloud3</tabstop>
  <tabstop>cb</tabstop>
  <tabstop>prob30</tabstop>
  <tabstop>prob40</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
